app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,47 @@
+class TasksController < ApplicationController
+  before_action :set_project, only: %i[index new create]
+  before_action :set_task, only: %i[show edit update create]
+
+  def index
+    @project.tasks = Task.includes(:project)
```

Comment:
Do like this, and use in `app/views/tasks/index.html.erb` not a @project.tasks just @tasks
```suggestion
    @tasks = @project.tasks
```